src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -122,17 +126,24 @@ private async Task<int> GetMaxProcessCount()
             return (await GetAllWorkerChannelsAsync()).Count();
         }
 
-        internal async Task InitializeJobhostLanguageWorkerChannelAsync()
+        internal async Task InitializeJobhostLanguageWorkerChannelAsync(IEnumerable<string> languages = null)
         {
-            await InitializeJobhostLanguageWorkerChannelAsync(0);
+            if (languages == null)
```

Comment:
Do you need to check if languages is not empty?

Reply:
That will never happen since its not user input, but i think its still good idea to check for empty also!